govtool/frontend/src/i18n/locales/en.ts

```diff
@@ -740,7 +740,7 @@ export const en = {
           "Looks like you cannot retire, because currently you are not a Direct Voter.",
       },
       registerDescription:
-        "A Direct Voter is someone that can vote on any Governance Action with their own Voting Power, which is equal to the balance of Ada in their connected wallet. <0>Learn More</0> about Direct Voter.\n\nBecoming a Direct Voter will require a refundable deposit equal to your voting power.\n\nYour deposit will be refunded if you either retire or delegate your voting power to someone else (a DRep)",
+        "A Direct Voter is someone that can vote on any Governance Action with their own Voting Power. They do not wish to campaign for delegations from others, although anyone can still delegate their voting rights to you if they know your DRep ID. <0>Learn More</0> about Direct Voter.\n\nOn-chain direct voters are not supported by the protocol but are instead a abstraction over the on-chain DReps. Direct voters are not expected to link and host metadata as DReps are.\n\nBecoming a Direct Voter will require a refundable deposit of ₳<strong>{{deposit}}</strong>. Your deposit will be refunded if you retire. You can decide to become a DRep at anytime if you are a direct voter you wont need to pay another deposit.",
```

Comment:
This sentence `They do not wish to campaign for delegations from others, although anyone can still delegate their voting rights to you if they know your DRep ID.` starts talking about Direct Voter in 3rd person, but then switches to 2nd person

typos: 
- `a abstraction`
- `wont`

If I understand correctly `Direct Voter` should always be capitalized?

Reply:
ooops!

good catch!

Reply:
fixed via [7e1344c](https://github.com/IntersectMBO/govtool/pull/2005/commits/7e1344c4ffc1a1e678588b4b39f88da84ecd249d)